gatsby-config.js

```diff
@@ -9,6 +9,20 @@ module.exports = {
     'MarkdownRemark.frontmatter.host': 'MarkdownRemark.frontmatter.location'
   },
   plugins: [
+    {
+      resolve: 'gatsby-plugin-manifest',
+      options: {
+        name: 'dsmJS',
+        short_name: 'dsmJS',
+        description: 'Des Moines JavaScript User Group',
+        lang: 'en',
+        start_url: '/',
+        background_color: '#EFDA4F',
+        theme_color: '#039998',
+        display: 'standalone',
+        icon: './static/dsmJS.svg'
```

Comment:
could this pull from a package, or does it need to be a local path? we already have this in the components repo (maybe thats where you pulled it in from). it would be nice if we could maintain a single copy

Reply:
i [recently switched over to this plugin](https://github.com/travi-org/matt.travi.org/commit/c1623ef7d97b50ddf197f358bd07a9298cefd665) for my personal site, and the unfortunate answer is [kinda](https://github.com/travi-org/matt.travi.org/blob/e1e5cc675b42042f5a9b87eb9ad7af26b2d954ab/gatsby-config.js#L53). 

i didn't find a way to resolve the path like node does so that it could be referenced relative to the package. seems unlikely to be a problem where the deduping algorithm would move it, but that is my hesitation.